### DISCUSSION TITLE
Clock 2.7

### DISCUSSION THREAD

**Discussion by BlackyHawky** [BlackyHawky/Clock#96] [Announcements]
**Improvements**
- Choose the default style for the time picker;
- Add a fade transition when browsing settings;
- Customize alarm animation display:
  - The color of each element can be changed;
  - Digital clock font size and alarm title font size can be customized;
- Add a "Nothing" action to the volume buttons;
- Add the ability to display the wallpaper when the phone is turned on at the end of the timer;
- Improve settings views:
  - The "Display card backgrounds" and "Display background borders" settings now apply to all views of the settings;
- Improve the code for the screensaver:
  - Fix some settings that were not hidden (not all Android versions were affected);
  - The second hand color can be changed for the analog clock;
- Improve views by reducing some margin;
- Update gradle, ndk and translations;
- Clean and improve code;
- Readme updated to show translation status;

<hr /><em>This discussion was created from the release <a href='https://github.com/BlackyHawky/Clock/releases/tag/2.7'>Clock 2.7</a>.</em>

**Comment by timofort** [comment]
Thank you Blacky. This release is exactly what I dreamed, with own phone home wallpaper as background. Excellent job, you can congratulate yourself to put the only clock android app with personal background, with Samsung OneUI, at this time.

**Reply by BlackyHawky** [reply]
Thank you very much for your message. ♥
I'm very happy to know that the hours I've spent trying to improve this application are satisfying a lot of people.
I hope it will be the same for future versions. 😉
